### PR TITLE
Fix some non-controversial findings

### PR DESCRIPTION
**Pull request by jtraglia**
* Move some checks to permanently disabled section.
* Fix some checks & remove them from "maybe fix section".

### CODE REVIEW

File: serialization/serialization_test.go [hunk 1]
```diff
@@ -67,16 +68,17 @@ func assertPolyEqual(t *testing.T, lhs, rhs kzg.Polynomial) {
 	}
 }
 
-// Assert that two polynomials are different -- differ at at least one
```
Comment (kevaundray) [inline serialization/serialization_test.go:70]: I think this is correct -- "they differ at atleast one evaluation"

File: serialization/serialization_test.go [hunk 2]
```diff
@@ -67,16 +68,17 @@ func assertPolyEqual(t *testing.T, lhs, rhs kzg.Polynomial) {
...
 // evaluation
 func assertPolyNotEqual(t *testing.T, lhs, rhs kzg.Polynomial) {
+	t.Helper()
 	polyLen := assertPolySameLength(t, lhs, rhs)
 
 	// element at index `i` in polyPredicate stores whether the
 	// evaluations at index `i` are the same
 	//
 	// We need this because two polynomials are different
-	// if they differ at at least one evaluation
```
Comment (kevaundray) [inline serialization/serialization_test.go:79]: same comment here

File: serialization/serialization_test.go (hunk 1 above)
Comment (jtraglia) [reply serialization/serialization_test.go:70]: Oh you're right. Mind if I rephrase it to something like "at least one evaluation differs"?

Comment (kevaundray) [reply serialization/serialization_test.go:70]: yep sure!

**Review by kevaundray — APPROVED** [review]
